GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh

```diff
@@ -211,6 +211,10 @@ function CreateMetaDistribution()
     eval $copyGitPkgCmd
 
     UpdateDistributionFile "$GITPKGNAME" "$GITVERSIONSTRING"
+
```

Comment:
I will probably break this out into a more appropriate function (or maybe have a function that pulls out the pkg from the dmg, and then we can create the meta-installer separately).

I also was going to look at having the git for mac build publish the .pkg directly, so we did not need to extract it from the dmg